qa/workunits/rbd/cli_migration.sh

```diff
@@ -39,6 +41,23 @@ create_base_image() {
     rbd bench --io-type write --io-pattern rand --io-size=4K --io-total 128M ${image}
 }
 
+create_base_image_with_namespace() {
+    local namespace=$1
+    local image=$2
+
+    # Create a new namespace and new image within that namespace
+    rbd namespace create rbd/${namespace}
```

Comment:
Both namespaces need to be removed in `cleanup()` -- it should be possible to run the script multiple times in a row without any manual cleanup in between.

Reply:
added `namespace` removal in in cleanup()